src/pages/uxp/reference-js/Global Members/HTML DOM/Element.md

```diff
@@ -184,6 +184,12 @@ Indicates if the element will focus automatically when it is loaded
 - https://w3c.github.io/DOM-Parsing/#the-innerhtml-mixin
 
 
+| Param | Type |
+| --- | --- |
+| value | `string` | 
+
+
+
```

Comment:
I'm not really happy with this, but we can create a separate issue for this if you want to track it that way, since it's a bigger issue, anyways (with the two separate declarations of `innerHTML`

Reply:
@kareemmyk ^^^

Reply:
@padmkris123 , @pklaschka .... there are 2 innerHTML.
One is a ```get``` and the other is a ```set```.
For now can you update the first one as ```get``` and the second one as ```set```.
I will see what would be the appropriate item for this. I am tracking this issue under https://jira.corp.adobe.com/browse/UXP-20643

Reply:
@pklaschka  please check